scripts/setup_host.sh

```diff
@@ -252,6 +252,21 @@ function ubu_install_swtpm() {
     cd -
 }
 
+ubu_update_bt_fw(){
+    if [ -d "linux-firmware" ] ; then
+            rm -rf linux-firmware
+    fi
+    git clone https://kernel.googlesource.com/pub/scm/linux/kernel/git/firmware/linux-firmware
+    cd linux-firmware
+    git checkout 73144e02dcff54529ab42d951de682740d204180
+    cd -
+    sudo cp linux-firmware/intel/ibt-19-0-4* /lib/firmware/intel
+    hciconfig hci0 up
+    hcitool cmd 3f 01 01 01 00 00 00 00 00
+    sleep 5
```

Comment:
Is it possible to check return value or status instead of sleep?

Reply:
hcitool cmd 3f 01 01 01 00 00 00 00 00 will return instantly but, firmware download process will take around 4 sec time. Hence i have added sleep for 5 sec